src/components/atoms/navbar.tsx

```diff
@@ -23,19 +24,38 @@ interface NavBarProps {
 }
 
 export const shareApp = async (t: TFunction) => {
+  const title = t('common:name');
+  const url = t('common:url');
+  const message = t('common:message');
+
   try {
-    await Share.share(
-      {
-        title: t('common:message'),
-        message:
-          Platform.OS === 'android' ? t('common:url') : t('common:message'),
-        url: t('common:url')
+    await Share.open(Platform.select({
+      ios: {
+        activityItemSources: [
+          {
+            placeholderItem: {
+              type: 'url',
+              content: icon
+            },
+            item: {
+              default: {
+                type: 'text',
+                content: `${message} ${url}`
+              },
+            },
+            linkMetadata: {
+               title: title,
+               icon: icon
+            }
+          },
+        ],
+      },
+      default: {
+        title: 'abc',
```

Comment:
Shouldn't the default title be `title` and not `"abc"`

Reply:
You're right, I just posted this to get some more in device testing